.vsts-ci/misc-analysis.yml

```diff
@@ -36,6 +36,11 @@ jobs:
     displayName: Install Pester
     condition: succeededOrFailed()
 
+  - bash: |
+      curl -o- --progress-bar -L https://yarnpkg.com/install.sh | bash
+    displayName: Bootstrap Yarn
+    condition: succeededOrFailed()
+
   - bash: |
       sudo npm install -g markdown-spellcheck@0.11.0
```

Comment:
If we are going to use yarn, can we use the yarn commands?

Reply:
Can’t install globally with yarn.  This needs to stay.

Reply:
Can you add a comment saying this here and on line 25 in the test?

Reply:
Okay